packages/web-react/src/components/FileUploader/__tests__/useFileQueue.test.ts

```diff
@@ -0,0 +1,55 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { useFileQueue } from '../useFileQueue';
+
+describe('useFileQueue', () => {
+  it('should return defaults', () => {
+    const { result } = renderHook(() => useFileQueue());
+
+    expect(result.current.fileQueue).toBeDefined();
+    expect(result.current.onDismiss).toBeDefined();
+    expect(result.current.addToQueue).toBeDefined();
+    expect(result.current.clearQueue).toBeDefined();
+  });
+
+  it('should add files to queue', () => {
+    const { result } = renderHook(() => useFileQueue());
+    const file1 = new File([''], 'test1.txt', { type: 'text/plain', lastModified: Date.now() });
```

Comment:
Should this be a current datetime? I suggest using just static string.

Reply:
This is how it is used in web packages tests, so I just used the same method to create a dummy file

Reply:
Ok, we are not working with this property anyway 🤷 But just for clarification: Date.now() is changing the value for every test. When using this property, the test is not repeatable because the value will not be the same.

So if we find some time we can refactor this to a static value in both packages.